setup.py

```diff
@@ -6,561 +6,58 @@
-                    if code != 0:
-                        raise Exception("Failed to build for python version " + python_version)
-                    if not os.path.isfile(package_path):
-                        try:
-                            print("files that DO exist: " + repr(os.listdir(os.path.basename(package_path))))
-                        except Exception as e:
-                            print(" (failed to list files that do exist, %s)" % str(e))
-                        raise Exception("conda said it would build %s but it didn't" % package_path)
-
-                _safe_makedirs(os.path.dirname(final_package_path))
-                print("Copying %s to %s" % (package_path, final_package_path))
-                shutil.copyfile(package_path, final_package_path)
-                print("Created %s" % final_package_path)
-
-            for arch in ('osx-64', 'linux-32', 'win-32', 'win-64'):
-                if arch == build_arch:
-                    continue
-                converted_output_dir = os.path.join(python_scoped_package_dir)
-                converted_package_path = os.path.join(converted_output_dir, arch, os.path.basename(package_path))
-                if os.path.isfile(converted_package_path):
-                    print("Already converted to %s from %s for python %s" % (arch, build_arch, python_version))
-                else:
-                    print("Creating %s by conversion %s=>%s" % (converted_package_path, build_arch, arch))
-                    _safe_makedirs(converted_output_dir)
-                    # this automatically creates the "arch" directory to put the package in
-                    code = subprocess.call(['conda', 'convert', '--platform', arch, final_package_path, '--output-dir',
-                                            converted_output_dir])
-                    if code != 0:
-                        raise Exception("Failed to convert from %s to %s to create %s" %
-                                        (build_arch, arch, converted_package_path))
-                    all_final_package_paths.append(converted_package_path)
-
-        print("Packages in " + self.packages_dir)
+def get_description():
+    """Get long description."""
+    with open(os.path.join(HERE, 'README.md'), 'r') as f:
+        data = f.read()
+    return data
 
 
 setup(name='anaconda-project',
-      version=VERSION,
-      author="Continuum Analytics",
-      author_email='info@continuum.io',
+      version=get_version(),
+      keywords=["conda anaconda project reproducible data science"],
       url='http://github.com/Anaconda-Server/anaconda-project',
-      description='Library to load and manipulate project directories',
       license='New BSD',
+      author="Anaconda, Inc",
+      author_email='info@anaconda.com',
+      maintainer='Anaconda, Inc',
+      maintainer_email='info@anaconda.com',
+      description='Library to load and manipulate project directories',
```

Comment:
I know it's not part of this PR... just a comment, this feels a bit unclear

Reply:
Yeaj that description is not great. Suggwstions on a new one?

Reply:
Something  like "Library to manage and run reproducible projects"? Not great as well... :-/

@pzwang suggestions?

Reply:
How about "Tool for encapsulating, running, and reproducing data science projects"